cli/config/configuration-fc.toml.in

```diff
@@ -78,6 +79,9 @@ default_memory = @DEFMEMSZ@
 # Default 0
 #memory_offset = 0
 
+# Specify whether cpu/memory used by guest is counted in container resource quota.
```

Comment:
Could you explain here when or how this should be enabled?

Reply:
Per @grahamwhaley 's comment, I'm planning to update the document to explain this, WDYT?

Reply:
ack

Reply:
Lets add in the documentation, I wanted to add a bit more of information but I probably add all the context in a one-line summary may be difficult.